app/code/Epam/DI/etc/module.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Module/etc/module.xsd">
+    <module name="Epam_DI" setup_version="0.0.1"/>
```

Comment:
Do you really need this setup_version attribute in this module?

Reply:
Doesn't need, thanks.